cli/src/main.rs

```diff
@@ -0,0 +1,62 @@
+use clap::Parser;
+use cli::command::delete::delete_command;
+use cli::command::install::install_command;
+use cli::command::select_system::select_system_command;
+use cli::command::software_list::software_command;
+use cli::command::system_list::systems_command;
+use cli::command::update::update_command;
+use cli::errors::Errors;
+use cli::{Args, CliConfig, Operation};
+use config::{Config, FileFormat};
+
+const CONFIG_NAME: &str = ".capstone_config.toml";
+
+fn main() -> Result<(), Errors> {
+    let home_dir = match dirs::home_dir() {
+        None => Err(Errors::CustomError(
+            "Failed to get home directory".to_string(),
+        )),
+        Some(path) => Ok(path),
+    }?;
```

Comment:
Minor comment
Looks like it can be simplified to 

```rust
let home_dir = dirs::home_dir().ok_or(Errors::CustomError("Failed to get home directory".to_string()))?;
```